src/ALCProxy.Communication/CommObjects.cs

```diff
@@ -0,0 +1,119 @@
+﻿using System;
+using System.Linq;
+using System.Reflection;
+using System.Runtime.Loader;
+
+namespace ALCProxy
+{
+
+    public interface IServerObject {
+       // object CallObject(MethodInfo method, object[] args);
+    }
+    public interface IClientObject
```

Comment:
Move these public types to their own separate files (especially for things like interfaces).